src/azure-cli/azure/cli/command_modules/role/custom.py

```diff
@@ -129,25 +130,38 @@ def _search_role_definitions(cli_ctx, definitions_client, name, scopes, custom_r
 
 
 def create_role_assignment(cmd, role, assignee=None, assignee_object_id=None, resource_group_name=None,
-                           scope=None, assignee_principal_type=None):
+                           scope=None, assignee_principal_type=None, description=None,
+                           condition=None, condition_version=None):
+    """Check parameters are provided correctly, then call _create_role_assignment."""
     if bool(assignee) == bool(assignee_object_id):
         raise CLIError('usage error: --assignee STRING | --assignee-object-id GUID')
 
     if assignee_principal_type and not assignee_object_id:
         raise CLIError('usage error: --assignee-object-id GUID [--assignee-principal-type]')
 
+    # If condition is set and condition-version is empty, condition-version defaults to "2.0".
+    if condition and not condition_version:
+        condition_version = "2.0"
```

Comment:
How about adding this default value in the help message ?

Reply:
Nice catch. Added.